package/MDAnalysis/analysis/encore/similarity.py

```diff
@@ -751,52 +751,40 @@ def hes(ensembles,
 
     Parameters
     ----------
-
     ensembles : list
         List of Universe objects for similarity measurements.
-
     selection : str, optional
         Atom selection string in the MDAnalysis format. Default is "name CA"
-
     cov_estimator : str, optional
         Covariance matrix estimator method, either shrinkage, `shrinkage`,
         or Maximum Likelyhood, `ml`. Default is shrinkage.
-
-    mass_weighted : bool, optional
-        Whether to perform mass-weighted covariance matrix estimation
-        (default is True).
-
+    weights : str/array_like, optional
```

Comment:
I mostly still want to add tests for custom weights. My current idea is to use the masses as custom weights and compare the outputs when I chose `weights='mass'` that should be sufficient.